tests/test_document.py

```diff
@@ -16,24 +36,18 @@ def teardown(self):
         self.ctx.pop()
         super(TestDocument, self).teardown()
 
-    def test_new(self):
+    @staticmethod
```

Comment:
Well, my python GUI always warns me that these methods can be static as we neither use anything from the class, nor from the instance.
If I broke the convention with this, I can turn this back and ignore those warnings.